homeassistant/components/flexit_bacnet/climate.py

```diff
@@ -83,6 +84,22 @@ async def async_update(self) -> None:
         """Refresh unit state."""
         await self.device.update()
 
+    @property
+    def extra_state_attributes(self) -> dict[str, Any]:
+        """Return device specific state attributes."""
+        return {
+            "air_filter_exchange_interval": self.device.air_filter_exchange_interval,
+            "air_temp_setpoint_away": self.device.air_temp_setpoint_away,
+            "air_temp_setpoint_home": self.device.air_temp_setpoint_home,
+        }
+
+    @property
+    def hvac_action(self) -> HVACAction | None:
+        """Return current HVAC action."""
+        if self.device.electric_heater:
+            return HVACAction.PREHEATING
```

Comment:
I don't know the product but is this pre-heating and not just heating?

Reply:
`preheating` would be that it needs to heat an element/air/something inside before it starts operating and heating the room at which point it would then switch to `heating`. Heating would be that it's actually heating to heat the room.

So in your case it seems `heating` would be the right pick as it's heating an element while circulating the heated air into the room and not `preheating` an element when their would be no air circulation.

Reply:
I agree. Let's change it to `heating`